packages/preview/udesc/0.1.0/typst.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "udesc"
```

Comment:
Unless this is an official package this will need some sort of qualifier, see our naming rules:
https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules